src/serialize/column.rs

```diff
@@ -291,14 +297,21 @@ where
     }
 
     let predicate = |x: &&Archetype| -> bool { !x.is_empty() && x.satisfies::<Q>() };
-    let mut seq = serializer.serialize_seq(Some(world.archetypes().filter(predicate).count()))?;
+    let mut seq = serializer.serialize_seq(Some(world.archetypes().filter(predicate).count() + 2))?;
```

Comment:
Changes to the serialization schema must be reflected in the module level documentation.

Reply:
Updated.